test/functional/adb-commands-e2e-specs.js

```diff
@@ -7,7 +7,7 @@ import { apiLevel, platformVersion, MOCHA_TIMEOUT } from './setup';
 import { fs, mkdirp } from 'appium-support';
 import temp from 'temp';
 
-
+const should = chai.should();
```

Comment:
the way that i usually get around lint errors here is simply:

```js
chai.should();
```

i.e., you don't need to assign it to a variable for it to do the thing you need it to do.